frontend/public/components/filter-toolbar.tsx

```diff
@@ -218,12 +218,8 @@ const FilterToolbar_: React.FC<FilterToolbarProps & RouteComponentProps> = (prop
 
   const clearAll = () => {
     updateRowFilterSelected(selectedRowFilters);
-    if (!hideNameFilter) {
-      updateNameFilter('');
-    }
-    if (!hideLabelFilter) {
```

Comment:
Do we still want this check since we support `hideLabelFilter`? (I'm not sure if it matters.)

Reply:
`hideLabelFilter` check made sense when we had the same label filter search param key for search page and list pages. But later we changed that and this should have been changed then, but I see that I missed out on it. Now there is no possibility of `Clear All` of list page showing up inside `Search`(thanks to hideToolbar) in which case it would clear all the selectors( we only wanted to clear row filters initially) we can allow it to clear everything.